boot.properties

```diff
@@ -0,0 +1,5 @@
+#http://boot-clj.com
+#Fri Mar 11 02:22:39 CET 2016
```

Comment:
This file was autogenerated, it's just a timestamp of generation, for documentation purposes.